readme.md

```diff
@@ -287,6 +287,63 @@ Arguments passed to the CLI will always take precedence over the configuration i
 
 Note that providing files on the CLI overrides the `files` option. If you've configured a glob pattern, for instance `test/**/*.test.js`, you may want to repeat it when using the CLI: `ava 'test/integration/*.test.js'`.
 
+### Using `ava.config.js`
+
+To use an `ava.config.js` file:
+
+ 1. It must be in the same directory as your `package.json`.
+ 2. Your `package.json` must not contain an `ava` property.
+
+The config file can use either ESM or CommonJS syntax. Both of these are valid:
+
+```js
+module.exports = {
+	// …
+};
+```
+
+```js
+export default {
+	// …
+};
+```
+
+The config file may also export a factory function, which will be called with an object containing a `projectDir` property. This function must return a config object, and cannot be a promise.
+
+```js
+const config = ({projectDir}) => {
+	if (projectDir === '...') {
```

Comment:
`projectDir` will be an absolute path, so it'd be good if the example didn't show dots which imply it's somehow relative.

Providing the directory is useful if you have share AVA config amongst projects, e.g. in a work setting or a monorepo. You could then differentiate between projects when necessary. Perhaps this is something we should cover in a recipe, outside of this PR.